### ISSUE TITLE
Failed to activate the linter-bootlint package

### ISSUE THREAD

**Issue by simonlacey** [AtomLinter/linter-bootlint#23]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.4.3
**System**: Mac OS X 10.11.2
**Thrown From**: [linter-bootlint](https://github.com/AtomLinter/linter-bootlint) package, v1.0.0

### Stack Trace

Failed to activate the linter-bootlint package

```
At Cannot find module 'atom-package-path'

Error: Cannot find module 'atom-package-path'
    at Module._resolveFilename (module.js:336:15)
    at Function.Module._resolveFilename (/Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (/Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.install (/Users/simonlacey/.atom/packages/linter-bootlint/node_modules/atom-package-deps/lib/main.js:27:12)
    at Object.activate (/Users/simonlacey/.atom/packages/linter-bootlint/lib/init.js:25:34)
    at Package.module.exports.Package.activateNow (/Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/package.js:183:20)
    at /Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/package.js:156:32
    at Package.module.exports.Package.measure (/Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/package.js:92:15)
    at /Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/package.js:149:26
    at Package.module.exports.Package.activate (/Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/package-manager.js:524:21)
    at /Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/package-manager.js:505:29
    at Config.module.exports.Config.transactAsync (/Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/package-manager.js:500:19)
    at PackageManager.module.exports.PackageManager.activate (/Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/package-manager.js:483:46)
    at AtomEnvironment.module.exports.AtomEnvironment.startEditorWindow (/Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:688:21)
    at module.exports (/Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/src/initialize-application-window.js:28:10)
    at setupWindow (file:///Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/static/index.js:86:5)
    at window.onload (file:///Users/simonlacey/Downloads/Atom.app/Contents/Resources/app.asar/static/index.js:41:9)
```

### Commands

```
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
atom-html-preview, v0.1.13
linter-bootlint, v1.0.0

# Dev
No dev packages
```

**Comment by Arcanemagus** [comment]
If you go to the package directory (`~/.atom/packages/linter-bootlint`) and run `apm install`, do you still see the issue?

**Comment by Arcanemagus** [comment]
This was due to a bug in a dependency that has since been fixed, you can either uninstall/reinstall this package, or run `apm rebuild` in the package's folder from a terminal to fix this. Sorry about that!

**Comment by simonlacey** [comment]
No worries thank you.

> Landon Abney <mailto:notifications@github.com>
> February 8, 2016 at 4:02 PM
>
> Closed #23 <https://github.com/AtomLinter/linter-bootlint/issues/23>.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/AtomLinter/linter-bootlint/issues/23#event-543417504>.
>

-- 
Regards,

Simon Lacey | Sr Web Designer
Simple Health | Simple Insurance Leads

email: slacey@sileads.com
phone: (754) 208-3573 ext: 2052
web: www.simpleinsuranceleads.com <http://www.simpleinsuranceleads.com>
